pkgs/components/src/icons/index.tsx

```diff
@@ -1,70 +1,71 @@
 import React from "react"
+import account from "@material-ui/icons/Person"
 import add from "@material-ui/icons/Add"
+import back from "@material-ui/icons/ArrowBack"
+import blog from "@material-ui/icons/Notes"
+import cancel from "@material-ui/icons/Cancel"
+import close from "@material-ui/icons/Close"
+import code from "@material-ui/icons/Code"
 import collapse from "@material-ui/icons/ExpandMore"
-import expandRight from "@material-ui/icons/ChevronRight"
+import dropdown from "@material-ui/icons/MoreVert"
+import edit from "@material-ui/icons/Edit"
+import email from "@material-ui/icons/Email"
 import expandDown from "@material-ui/icons/ExpandMore"
+import expandRight from "@material-ui/icons/ChevronRight"
+import gitHub from "@material-ui/icons/GitHub"
+import help from "@material-ui/icons/HelpOutline"
 import home from "@material-ui/icons/Home"
-import openModal from "@material-ui/icons/OpenInNew"
+import linkedIn from "@material-ui/icons/LinkedIn"
 import menu from "@material-ui/icons/Menu"
-import schedule from "@material-ui/icons/Schedule"
+import menuBook from "@material-ui/icons/MenuBook"
+import openModal from "@material-ui/icons/OpenInNew"
 import run from "@material-ui/icons/PlayArrow"
-import edit from "@material-ui/icons/Edit"
-import trash from "@material-ui/icons/Delete"
-import back from "@material-ui/icons/ArrowBack"
-import help from "@material-ui/icons/HelpOutline"
-import account from "@material-ui/icons/Person"
-import dropdown from "@material-ui/icons/MoreVert"
-import view from "@material-ui/icons/RemoveRedEye"
 import save from "@material-ui/icons/Done"
-import close from "@material-ui/icons/Close"
-import linkedIn from "@material-ui/icons/LinkedIn"
-import email from "@material-ui/icons/Email"
-import gitHub from "@material-ui/icons/GitHub"
+import schedule from "@material-ui/icons/Schedule"
+import settings from "@material-ui/icons/Settings"
+import trash from "@material-ui/icons/Delete"
 import twitter from "@material-ui/icons/Twitter"
 import video from "@material-ui/icons/VideocamOutlined"
-import blog from "@material-ui/icons/Notes"
-import code from "@material-ui/icons/Code"
-import settings from "@material-ui/icons/Settings"
-import cancel from "@material-ui/icons/Cancel"
+import view from "@material-ui/icons/RemoveRedEye"
 import SvgIcon, { SvgIconProps } from "@material-ui/core/SvgIcon"
 import { twitchPath } from "./paths"
 
-const toIcon = (path: string) => (props: SvgIconProps) =>
-    (
-        <SvgIcon {...props}>
-            <path d={path} />
-        </SvgIcon>
-    )
+const toIcon = (path: string) => (props: SvgIconProps) => (
```

Comment:
Maybe also formatter related? Not sure why the indentation changed here. Don't have a strong preference either way as long as my formatter doesn't try to change it back next time I try to edit the file.